fem/bilininteg.hpp

```diff
@@ -44,6 +44,9 @@ class BilinearFormIntegrator : public NonlinearFormIntegrator
        used later in the methods AddMultPA() and AddMultTransposePA(). */
    virtual void AssemblePA(const FiniteElementSpace &fes);
 
+   /// Assemble diagonal into Vector @a diag.
+   virtual void AssembleDiagonalPA(Vector &diag) const;
```

Comment:
Is this what this does, or is it "Assemble diagonal and add it to Vector @a diag."?

Reply:
You're right, I'll clarify the documentation.

Reply:
Thank you Andrew. Can you take a look at the Travis errors?